### ISSUE TITLE
LeftPanel keyboard navigation often doesn't work if you use the mouse to navigate to the filter box

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#12656]
If you click on the filter box, start typing, and then try to use arrow keys to navigate the filtered LeftPanel it often doesn't work. If you navigate there via cmd/ctrl-K it gets it right however

**Comment by Infinity-Intellect** [comment]
I'm new to the community and would like to work on this issue. 
I followed your steps to reproduce the issue, however I am able to navigate to a particular filtered chat room using my arrow keys. Is there any other way specific way by which I can reproduce the issue ?

**Comment by turt2live** [comment]
It's possible to be a macOS specific issue. I also can't reproduce.

**Comment by aaronraimist** [comment]
I haven't been able to reproduce this in either. Tried both Firefox and Chrome on macOS.

**Comment by turt2live** [comment]
This is fixed in the new room list anyhow
